themes/10up-theme/includes/blocks.php

```diff
@@ -69,6 +69,7 @@ function register_theme_blocks() {
 
 					// create helpful variables that will be accessible in markup.php file
 					$context = $block->context;
+					$wrapper_attributes = wp_kses_post( get_block_wrapper_attributes() );
```

Comment:
@devinle We actually had this here previously but decided to move the `get_block_wrapper_attributes` into the actual markup.php files because you need to be able to pass properties to the `get_block_wrapper_attributes` function. 

Also it should not be escaped in that way. See #108

Reply:
Done!